src/cloud/createComment.js

```diff
@@ -0,0 +1,24 @@
+import ExtendableError from 'extendable-error-class';
+import FeedService from '../services/FeedService';
+
+class CreateCommentError extends ExtendableError {}
+
+const createConnection = async request => {
+  const commentData = request.params;
```

Comment:
I prefer it t be destructured

Reply:
To create a new object instead of using the original?

Reply:
Just code style